SaintCoinach/ex.json

```diff
@@ -2718,63 +2990,63 @@
           }
         },
         {
-          "index": 34,
+          "index": 33,
```

Comment:
I think these offsets are off by one because of the update - would you mind fixing that?